distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc

```diff
@@ -1,3 +1,230 @@
+** *View Sharing*: View and edit settings for sharing this workspace. (User must be signed in to share or view shared workspaces.)
+** *View Details*: View the current details for a cloud-based workspace (User must be signed in).
+** *Duplicate*: Create a copy of this workspace.
+** *Subscribe*: Receive updates on this workspace.
+** *Move to Trash*: Delete (archive) this workspace.
+
+==== Sharing Workspaces
+
+In the ${catalog-ui}, workspaces can be shared between users at different levels of access as needed.
+
+.Share a Workspace
+. Select the Workspace.
+. Select the *Options* menu.
+. Select *View Sharing*.
+.. To share by user role, set the drop-down menu to *Can Access* for each desired role. All users with that role will be able to view.
+.. To share with an individual user, add their email to the email list.
+. Click *Apply*.
+
+.Remove Sharing on a Workspace
+. Select the Workspace.
+. Select the *Options* Menu.
+. Select *View Sharing*.
+.. To remove the workspace from users with specific roles, set the drop-down menu to *No Access* for that role.
+.. To remove individual users, remove their email from the email list.
+
+=== Searching with ${catalog-ui}
+
+The Search pane has three tabs: *Searches*, *Results*, and *Bookmarks*.
+
+==== Searches Tab
+
+View, edit, and add searches from this tab.
+
+.Search Menu Options
+* *Run*: Trigger this search to begin immediately.
+* *Stop*: Stop this search.
+* *Delete*: Remove this search
+* *Duplicate*: Create a copy of this search as a starting point
+* *Search Archived*: Execute the search, but specifically for archived results.
+
+===== Editing a Standing Search
+
+Existing searches and their settings can be updated by clicking on the search in the *Searches* tab of the workspace.
+
+* *Basic*
+* *Advanced*
+* *Settings*
+* *Notifications*
+* *Status*
+
+====== Editing Search Settings
+
+Under the settings tab for each search, edit the default *Sorting* and *Federation* (which sources are searched).
+
+====== Setting Search Frequency
+
+Edit the *Notifications* tab for a search to set the interval at which to run the query.
+
+====== Viewing Search Status
+
+The *Status* tab on the search preview displays the creation date and the sources successfully searched.
+
+==== Viewing Search Results
+
+Returned search results can be refined further, bookmarked, and/or downloaded from the *Results* tab.
+Results sets are color-coded by source as a visual aid.
+There is no semantic meaning to the colors assigned.
+
+. Select a saved query.
+. Perform any of these action upon the results list of the selected query:
+.. Filter results further.
+.. Re-sort list(default is _title_/_ascending_).
+.. Toggle between *list* and *gallery* view.
+
+==== Bookmarks Tab
+
+Search Results that have been bookmarked can be filtered and sorted.
+
+.Bookmarked Result Options
+* *Add to Bookmarks*.
+** Changes to *Remove from Bookmarks* for bookmarked items.
+* *Hide from Future Searches*.
```

Comment:
Recommend adding description of what these options do here: 

- Remove from Bookmarks - `Removes the result from your bookmarks.`
- Hide from Future Searches - `Adds to a list of results that will be hidden from future searches. To clear this list, go to the toolbar, click Edit, and then click reset blacklist.`
- Expand Metacard View - `Navigates to a view that only focuses on this particular result.` 
- Download - `Downloads the result's associated product directly to your machine.`